jaxb-ri/codemodel/codemodel/src/main/java/com/sun/codemodel/JFormatter.java

```diff
@@ -267,6 +302,17 @@ public JFormatter t(JClass type) {
         return this;
     }
 
+    private String renameClassName(String fullName) {
+        if (classNameReplacer != null) {
```

Comment:
if this property gets initialized to `Collections.emptyMap()`, the null check can go away

Reply:
Done